code/range/Model/src/main/java/ru/unn/agile/range/model/IntegerSet.java

```diff
@@ -0,0 +1,34 @@
+package ru.unn.agile.range.model;
+
+import java.util.StringJoiner;
+import java.util.stream.Stream;
+
+public class IntegerSet {
+    private int[] values;
+
+    @Override
+    public String toString() {
+        StringJoiner stringJoiner = new StringJoiner(",");
+        for (int value : values) {
+            stringJoiner.add(String.valueOf(value));
+        }
+        return '{' + stringJoiner.toString() + '}';
+    }
+
+    public IntegerSet(final String setString) {
+        String trimSetString = setString.trim();
+        trimSetString = trimSetString.substring(1, trimSetString.length() - 1);
+        trimSetString = trimSetString.trim();
+        String[] set = trimSetString.split("[,\\s]+");
+        values = Stream.of(set).mapToInt(Integer::parseInt).toArray();
+    }
+
+    public IntegerSet(final int[] values) {
+        this.values = values;
+    }
+
+    public int[] getValues() {
+        return values;
+    }
+
+}
```

Comment:
By the way, I don't get why do you need IntegerSet class. 
You can use just List<Integer> or int[] instead of your personal class.
Cause, I don't see any special logic in that class.

Reply:
IntegerSet uses in more than one cases and I think that put this functionality to a separate class